### ISSUE TITLE
Tests github.com/kubernetes-incubator/cri-o/server pkg

### ISSUE THREAD

**Issue by runcom** [kubernetes-sigs/cri-o#893]
Also maybe split those huge functions in pieces to be easily testable. Any taker? :angel: 

```
ok  	github.com/kubernetes-incubator/cri-o/server	0.116s	coverage: 0.8% of statements
```

**Comment by weiwei04** [comment]
Me and @willstudy would like to help make server pkg more easily testable and add ut :)

**Comment by rhatdan** [comment]
@weiwei04 @runcom Did this ever happen?

**Comment by saschagrunert** [comment]
This should be solved via #1992

**Comment by umohnani8** [comment]
Fixed in #1992
